docs/administration-guide/06-collections.md

```diff
@@ -1,35 +1,61 @@
 ## Creating Collections for Your Saved Questions
 ---
 
-Collections are a great way to organize your saved questions and decide who gets to see and edit things. Collections could be things like, "Important Metrics," "Marketing KPIs," or "Questions about users." Multiple [user groups](05-setting-permissions.md) can be given access to the same collections, so we don't necessarily recommend naming collections after user groups.
+![Collection detail](images/collections/collection-detail.png)
+
+Collections are a great way to organize your saved questions and decide who gets to see and edit things. Collections could be things like, "Important Metrics," "Product Team," "Marketing KPIs," or "Questions about users." Collections can even contain other collections, allowing you to create an organizational structure that fits your team. You can also choose which user groups should have what level of access to your collections (more on that below).
```

Comment:
We should probably update this first sentence to reflect the fact that you can organize more than just questions in collections.